src/exchange_solver.cc

```diff
@@ -3,11 +3,25 @@
 #include <vector>
 #include <map>
 
+#include "context.h"
 #include "exchange_graph.h"
 
 namespace cyclus {
 
-double ExchangeSolver::PseudoCost(double cost_add) {
+double ExchangeSolver::Cost(const Arc& a, bool exclusive_orders) {
+  return (exclusive_orders && a.exclusive()) ?
```

Comment:
That's correct (and expected). The exclusive orders flag says whether or not exclusive orders are allowed. It is true by default in both the XML reader and now the codebase. A user turning it off means that a user wants to only allow partial orders. Note that partial orders are a superset of exclusive orders (i.e., they include exclusive orders quantities). This flag is an optimization in the sense that it turns a MILP into an LP.